doc/quickstart-guide.md

```diff
@@ -0,0 +1,230 @@
+
+To do so verify that the `readyReplicas` field of the Kubernetes Deployment
+associated to the APIcast object is 1 or wait until it is:
+
+```
+$ echo $(kubectl get deployment apicast-example-apicast -o jsonpath='{.status.readyReplicas}')
+1
+```
+
+#### Verify APIcast gateway is running and available
+
+To verify that APIcast gateway is running and available you can port-forward
+the Kubernetes Service exposed by APIcast to your local machine and perform
+a test request.
+
+To do so:
+
+Port-forward the APIcast Kubernetes Service to localhost:8080
+```
+kubectl port-forward svc/apicast-example-apicast 8080
+```
+
+Then you can make a request to one of the 3scale Services that have been
+configured in 3scale and verify that a successful HTTP response with the expected
+contents of the configured backend for the corresponding specific service that
+is being checked.
+
+For example, if the service that has been created has been configured with
+the host "myhost.com" then the following can be executed:
+
+```
+$  curl 127.0.0.1:8080/test -H "Host: myhost.com"
+```
+
+A successful HTTP response should be received with the expected content
+that the Service backend should provide.
+
+### Providing a configuration Secret
+
+Set the following content when creating the APIcast object:
+
+```
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIcast
+metadata:
+  name: example-apicast
+spec:
+  embeddedConfigurationSecretRef:
+    name: asecretname
+```
+
+The ```spec.embeddedConfigurationSecretRef.name``` must be the name of an
+already existing Kubernetes secret that contains the configuration of the
+gateway. For more information about the contents of the secret and the
+configuration of the gateway see the
+[Embedded configuration secret](apicast-crd-reference.md#EmbeddedConfSecret) reference.
+
+An example of an embedded configuration secret that configures a 3scale service
+with the hostname "localhost" with the
+[3scale echo API](https://github.com/3scale/echo-api/) as the corresponding
+backend of the 3scale service:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: asecretname
+type: Opaque
+stringData:
+  config.json: |
+    {
+      "services": [
+        {
+          "proxy": {
+            "policy_chain": [
+              { "name": "apicast.policy.upstream",
+                "configuration": {
+                  "rules": [{
+                    "regex": "/",
+                    "url": "http://echo-api.3scale.net"
+                  }]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+```
+
+After creating the APIcast object you should verify that the APIcast pod is
+running and ready.
+
+To do so verify that the `readyReplicas` field of the Kubernetes Deployment
+associated to the APIcast object is 1 or wait until it is:
+
+```
+$ echo $(kubectl get deployment apicast-example-apicast -o jsonpath='{.status.readyReplicas}')
+1
+```
+
+#### Verify APIcast gateway is running and available
```

Comment:
pretty close! 

it would be nice if I could test just apicast (without needing some 3scale installation / configuration, as these docs explain... ;) )

Reply:
hi @gsaslis you don't need a 3scale installation (porta) if you directly provide a configuration file as explained.
Look at the "Providing a configuration Secret" in that same file and the "Verify APIcast gateway is running and available" subsection of that section. Is that what you want?

Reply:
@miguelsorianod thanks for the direct link - I missed that part before. 

Yes, this seems pretty close to what I had in mind (seeing as there's nothing extra for me to setup and I can use the 3scale echo API service). 

Thanks! I think this fixes #44